File: CP website/Work@tech/Arrays/Even Number of Digits.java
//https://workat.tech/problem-solving/practice/even-number-of-digits
class Solution {
	List<Integer> getEvenDigitNumbers (int[] arr) {
		// add your logic here
		ArrayList<Integer> l = new ArrayList<>();
		for(int i=0;i<arr.length;i++){
			if( (arr[i]>9 && arr[i]<100) || (arr[i]>999 && arr[i]<10000) || 
			   (arr[i]>99999 && arr[i]<1000000) ){
				l.add(arr[i]);
			}
		}
		
		return l;
	}
}

File: CP website/WorkatTech/Arrays/EvenNumberofDigits.py
# https://workat.tech/problem-solving/practice/even-number-of-digits
class Solution:
	def getEvenDigitNumbers(self, arr: List[int]) -> Optional[List[int]]:
		# add your logic here
		n  = len(arr)
		even = []
		for i in range(n):
			s  = str(arr[i])
			if(len(s)%2==0):even.append(arr[i])
		return even

